.github/workflows/release-please.yml

```
@@ -22,18 +22,28 @@ jobs:
     runs-on: ubuntu-latest
     needs: release-please
     if: needs.release-please.outputs.releases_created
+    strategy:
+      matrix:
+        cpu_arch:
+          - x86-64
+        fedora_release:
+          - 38
     container: 
       image: fedora:38
       options: --privileged
     steps:
       - uses: actions/checkout@v3
+      - name: Calculate ISO name
+        id: iso-name
+        run: |
+          echo "iso_name=universalblue-${{ matrix.fedora_release }}-${{ matrix.cpu_arch }}-$(date +%Y%m%d).iso" > $GITHUB_OUTPUT
```

```suggestion
          echo "iso_name=universalblue-${{ matrix.fedora_release }}-${{ matrix.cpu_arch }}-$(date +%Y%m%d)" > $GITHUB_OUTPUT
```
`.iso` is attached within the action

The rest LGTM